Apromore-Custom-Plugins/About-Portal-Plugin/src/main/resources/about/zul/about.zul

```diff
@@ -0,0 +1,50 @@
+<!--
+  #%L
+  This file is part of "Apromore Core".
+  Copyright (C) 2012 Felix Mannhardt.
+  Copyright (C) 2011 - 2017 Queensland University of Technology.
+  %%
+  Copyright (C) 2018 - 2021 Apromore Pty Ltd.
+  %%
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Lesser General Public License as
+  published by the Free Software Foundation, either version 3 of the
+  License, or (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Lesser Public License for more details.
+
+  You should have received a copy of the GNU General Lesser Public
+  License along with this program.  If not, see
+  <http://www.gnu.org/licenses/lgpl-3.0.html>.
+  #L%
+  -->
+
+<window id="pluginInfo" height="70vh" width="70vw" sizable="false" sclass="ap-about-window">
+  <tabbox vflex="1">
+    <tabs>
+      <tab label="Details" />
+    </tabs>
+    <tabpanels>
+      <tabpanel hflex="1">
+        <vlayout sclass="ap-about-details">
+          <div sclass="ap-about-logo">
+            <image src="/zkau/web/themes/${labels.theme}/common/img/brand/logo-white.svg" width="50%" style="position: relative; top: 3px"/>
+          </div>
+          <div sclass="ap-edition">${arg.edition}</div>
+          <div sclass="ap-version">Version: ${arg.version}</div>
+          <div sclass="ap-copyright">Copyright 2019-2021, ${arg.holder}</div>
```

Comment:
Bump the end date to 2022, I guess?

Reply:
Thanks! I think we need an annual bumping operation every time we switch to new year. The copyright is still 2021.